### DISCUSSION TITLE
getting error when deploying the application

### DISCUSSION THREAD

**Discussion by v10o** [cypress-io/cypress-realworld-app#1237] [General]
'''
yarn dev
yarn run v1.22.18
$ yarn db:seed:dev
$ yarn predev:cognito:ci
$ yarn copy:mock:awsexports && yarn copy:mock:awsexportses5
$ ncp scripts/mock-aws-exports.js src/aws-exports.js
$ ncp scripts/mock-aws-exports-es5.js aws-exports-es5.js
$ ncp ./data/database-seed.json ./data/database.json
$ cross-env NODE_ENV=development concurrently yarn:start:react yarn:start:api:watch
$ nodemon --exec yarn tsnode --watch 'backend' backend/app.ts
$ react-scripts -r @cypress/instrument-cra start
[start:api:watch] [nodemon] 2.0.15
[start:api:watch] [nodemon] to restart at any time, enter `rs`
[start:api:watch] [nodemon] watching path(s): backend/**/*
[start:api:watch] [nodemon] watching extensions: ts,json
[start:api:watch] [nodemon] starting `yarn tsnode backend/app.ts`
$ nyc --silent ts-node -P tsconfig.tsnode.json backend/app.ts
[start:react] [HPM] Proxy created: [ '/login', '/callback', '/logout', '/checkAuth', 'graphql' ]  ->  http://localhost:undefined
[start:react] [HPM] Subscribed to http-proxy events:  [ 'error', 'close' ]
[start:react] ℹ ｢wds｣: Project is running at http://192.168.1.71/
[start:react] ℹ ｢wds｣: webpack output is served from 
[start:react] ℹ ｢wds｣: Content not from webpack is served from /home/hamim/Desktop/Projects/cypress-realworld-app/public
[start:react] ℹ ｢wds｣: 404s will fallback to /
[start:react] Starting the development server...
[start:react] 
[start:react] Error: error:0308010C:digital envelope routines::unsupported
[start:react]     at new Hash (node:internal/crypto/hash:67:19)
[start:react]     at Object.createHash (node:crypto:130:10)
[start:react]     at module.exports (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/util/createHash.js:135:53)
[start:react]     at NormalModule._initBuildHash (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:417:16)
[start:react]     at handleParseError (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:471:10)
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:503:5
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:358:12
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/loader-runner/lib/LoaderRunner.js:373:3
[start:react]     at iterateNormalLoaders (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
[start:react]     at iterateNormalLoaders (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
[start:react] /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/react-scripts/scripts/start.js:19
[start:react]   throw err;
[start:react]   ^
[start:react] 
[start:react] Error: error:0308010C:digital envelope routines::unsupported
[start:react]     at new Hash (node:internal/crypto/hash:67:19)
[start:react]     at Object.createHash (node:crypto:130:10)
[start:react]     at module.exports (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/util/createHash.js:135:53)
[start:react]     at NormalModule._initBuildHash (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:417:16)
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:452:10
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/webpack/lib/NormalModule.js:323:13
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/loader-runner/lib/LoaderRunner.js:367:11
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/loader-runner/lib/LoaderRunner.js:233:18
[start:react]     at context.callback (/home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
[start:react]     at /home/hamim/Desktop/Projects/cypress-realworld-app/node_modules/react-scripts/node_modules/babel-loader/lib/index.js:59:103 {
[start:react]   opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
[start:react]   library: 'digital envelope routines',
[start:react]   reason: 'unsupported',
[start:react]   code: 'ERR_OSSL_EVP_UNSUPPORTED'
[start:react] }
[start:react] 
[start:react] Node.js v17.1.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[start:react] yarn run start:react exited with code 1
[start:api:watch] express-validator: sanitize(), sanitizeBody() and other sanitization-only middlewares have been deprecated.
[start:api:watch] Please use check(), body() and others instead, which must offer the same API, and more.
'''

**Comment by robertguss** [comment]
Try running `yarn start` and using Node 16. Or the LTS version of Node.

**Reply by v10o** [reply]
I have switched to node LTS 16 it is working now

**Reply by vaibhavkr-dev** [reply]
I am getting error with node LTS 16 on develop branch after running yarn start or yarn dev

$ yarn db:seed:dev
$ yarn predev:cognito:ci
$ yarn copy:mock:awsexports && yarn copy:mock:awsexportses5
$ ncp scripts/mock-aws-exports.js src/aws-exports.js
$ ncp scripts/mock-aws-exports-es5.js aws-exports-es5.js
$ ncp ./data/database-seed.json ./data/database.json
$ cross-env NODE_ENV=development concurrently yarn:start:react yarn:start:api:watch
[start:react] yarn run start:react exited with code 1
[start:api:watch] yarn run start:api:watch exited with code 1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by subiqa** [comment]
No luck on my end as well. 

'ncp' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
